django/db/backends/sqlite3/schema.py

```diff
@@ -55,8 +55,18 @@ def quote_value(self, value):
         else:
             raise ValueError("Cannot quote parameter value %r of type %s" % (value, type(value)))
 
+    def _is_referenced(self, table_name, column_name=None):
+        with self.connection.cursor() as cursor:
+            for other_table in self.connection.introspection.get_table_list(cursor):
+                constraints = self.connection.introspection._get_foreign_key_constraints(cursor, other_table.name)
+                for constraint in constraints.values():
+                    if (constraint['foreign_key'][0] == table_name and
```

Comment:
I would consider tuple unpacking in the line before: `constraint_table, constraint_column = constraint['foreign_key']`.